src/event/kqueue.rs

```diff
@@ -99,6 +99,12 @@ impl Event {
         self.inner.udata as _
     }
 
+    /// Get the raw data for this event.
+    pub fn data(&self) -> isize {
+        // On openbsd, data is an i64 and not an isize
```

Comment:
It looks like netbsd and newer versions of freebsd have this too. If this is sometimes an i64, should we cast to i64 here instead of isize?